capitoli/Nono_Passo_Liste.rst

```diff
@@ -218,19 +219,19 @@ La funzione *len* applicata ad una lista produce il numero di elementi di una li
 	i= 0
 	while i < len(allievi): 
 		print (allievi[i]
-	i= i + 1
-	Luigi Marco Filippo Paola Gabriella
+		i= i + 1
+	#Risultato: Luigi Marco Filippo Paola Gabriella
```

Comment:
Andrei a capo prima e metterei:
`# Risultato Atteso: bla bla`
Fai così in tutti i casi di blocco di codice con risultato annesso così teniamo un minimo di consistenza